code/modules/admin/verbs/randomverbs.dm

```diff
@@ -510,30 +510,64 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(!holder)
 		src << "Only administrators may use this command."
 		return
-	var/input = sanitize(input(usr, "Please enter anything you want. Anything. Serious.", "What?", "") as message|null, extra = 0)
-	var/customname = sanitizeSafe(input(usr, "Pick a title for the report.", "Title") as text|null)
-	if(!input)
-		return
-	if(!customname)
-		customname = "NanoTrasen Update"
+	var/reporttitle
+	var/reportbody
+	var/reporter
+	var/reporttype = input(usr, "Choose whether to use a template or custom report.", "Create Command Report") in list("Template", "Custom", "Cancel")
+	switch(reporttype)
+		if("Template")
+			establish_db_connection(dbcon)
+			if (!dbcon.IsConnected())
+				src << "<span class='notice'>Unable to connect to the database.</span>"
+				return
+			var/DBQuery/query = dbcon.NewQuery("SELECT title, message FROM ss13_ccia_general_notice_list WHERE deleted_at IS NULL")
+			query.Execute()
+
+			var/list/template_names = list()
+			var/templates[] = list()
+
+			while (query.NextRow())
+				template_names += query.item[0]

```

Comment:
My mistake on mixing languages. DM indexes start at one, so rename both `query.item[0]`s to `query.item[1]` and all `query.item[1]` to `query.item[2]`.